website/blog/2022-09-28-Data-processing-with-Spark-time-traveling.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Data processing with Spark: time traveling"
+authors:
+- name: Petrica Leuca
+category: blog
+image: /assets/images/blog/2022-09-28_Data_processing_with_Spark_time_traveling.png
```

Comment:
Refers to the image checked in from this pr - https://github.com/apache/hudi/pull/6972